src/tags.json

```
@@ -9,5 +9,7 @@
   "salmon": "Spatial data analysed with Salmon",
   "codex": "CODEX data",
   "slide-seq": "Slide-seq data",
-  "files": "Work with HuBMAP data files"
+  "files": "Work with HuBMAP data files",
+  "hra": "Human Reference Atlas data",
+  "sc": "Single cell analysis"
```

```suggestion
  "sc": "Single cell analysis",
  "R": "Template written in R"
```
Not sure what the right language is, but we should add the R tag as well.

---

good point! I want to add a github workflow soon to check whether any new tags have been added and to prompt someone who adds a new tag to actually add it to this list